.gitlab-ci.yml

```diff
@@ -52,7 +48,6 @@ workflow:
     - if: $CI_PIPELINE_SOURCE == "schedule"
     - if: $CI_COMMIT_REF_NAME == "master"
     - if: $CI_COMMIT_REF_NAME == "tags"
-    - if: $CI_COMMIT_BRANCH
```

Comment:
What does it imply that this variable is removed?

Reply:
nothing really, it's supposed to launch CI pipelines on every branch which is not desirable, we run the usual pipelines only on PR branches, it's covered by the next line. Besides, branches other than PR ones are not getting mirrored to GitLab.